app/policies/enrollment_policy.rb

```diff
@@ -0,0 +1,24 @@
+class EnrollmentPolicy < ApplicationPolicy
+  def index?
+    user_in_record_organization?
+  end
+
+  class Scope
+    attr_reader :user, :scope
+
+    def initialize(user, scope)
+      @user = user
+      @scope = scope
+    end
+
+    def resolve
+      return scope.all if user.global_role?
+
+      scope.joins(group: :chapter).where(group: { chapters: { organization_id: user.membership_organizations } })
```

Comment:
Does this work? I'd think that `organization_id` would expect ids, but `membership_organizations` are full on Organization objects. Maybe ActiveRecord is smart enough to figure. If it works fine, leave it.